### PR TITLE
add runtimejob handling additional apis

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Related to: #1163

### Details and comments
Adding runtime job to the job:
```
/api/v1/jobs/{job id}/add_runtimejob
```
list the runtime jobs created in the job:
```
/api/v1/jobs/{job id}/list_runtimejob
```

### CODE REVIEW

File: gateway/api/views.py [hunk 1]
```diff
@@ -347,6 +347,38 @@ def stop(self, request, pk=None):  # pylint: disable=invalid-name,unused-argumen
...
+        self, request, pk=None
+    ):  # pylint: disable=invalid-name,unused-argument
+        """Add RuntimeJob to job"""
+        tracer = trace.get_tracer("gateway.tracer")
+        ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
+        with tracer.start_as_current_span("gateway.job.add_runtimejob", context=ctx):
+            job = self.get_object()
+            runtimejob = RuntimeJob(
+                job=job,
+                runtime_job=request.data.get("runtime_job"),
```
Comment (IceKhan13) [inline gateway/api/views.py:361]: Should we check that `runtime_job` got passed in request somehow to display user friendly error? 

Something like
```python
if not request.data.get("runtime_job"):
    return Response({"message": "Got empty `runtime_job` field. Please, specify `runtime_job`."}, code / status=400)
```

File: gateway/api/views.py [hunk 2]
```diff
@@ -347,6 +347,38 @@ def stop(self, request, pk=None):  # pylint: disable=invalid-name,unused-argumen
...
+    @action(methods=["GET"], detail=True)
+    def list_runtimejob(
+        self, request, pk=None
+    ):  # pylint: disable=invalid-name,unused-argument
+        """Add RuntimeJpb to job"""
+        tracer = trace.get_tracer("gateway.tracer")
+        ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
+        with tracer.start_as_current_span("gateway.job.stop", context=ctx):
+            job = self.get_object()
+            runtimejobs = RuntimeJob.objects.all().filter(job=job)
```
Comment (IceKhan13) [inline gateway/api/views.py:376]:
```suggestion
            runtimejobs = RuntimeJob.objects.filter(job=job)
```

**Comment by akihikokuroda** [comment]
@IceKhan13 Thanks for review.  I updated both places.

**Review by IceKhan13 — APPROVED** [review]
Thank you @akihikokuroda !

**Pull request merged**